packages/account/src/Sections/Profile/PersonalDetails/validation.ts

```diff
@@ -62,21 +62,27 @@ const getBaseSchema = () =>
 export const getPersonalDetailsInitialValues = (
     account_settings: GetSettings,
     residence_list: ResidenceList,
-    states_list: StatesList
-) => {
-    const initialValues: GetSettings = {
-        first_name: account_settings.first_name,
-        last_name: account_settings.last_name,
-        phone: account_settings.phone,
-        date_of_birth: account_settings.date_of_birth,
+    states_list: StatesList,
+    is_virtual: boolean
+): GetSettings => {
+    const virtualAccountInitialValues: GetSettings = {
+        email_consent: account_settings.email_consent ?? 0,
```

Comment:
Hi @yauheni-deriv , 
I was curious about one thing why we are not using `true` or `false` instead of `0` or `1`.
Its not a blocker 😁   I was just thinking why not `true` or `false` 😁

Reply:
hi @fasihali-deriv , this is a good question.
not sure why it so))
this is how it comes from BE and how we send it to BE as well :)

Reply:
Alright. 
Thank you @yauheni-deriv 😁